doc/abrt-action-install-debuginfo.txt

```diff
@@ -30,17 +33,25 @@ OPTIONS
    Path to tmp directory. Default: /tmp/abrt-tmp-debuginfo-RANDOM_SUFFIX
 
 --cache::
-   Path to cache directory. Default: /var/cache/abrt-di
+   Path to cache directory. Default: /var/cache/abrt-dir
 
 --size_mb::
    Default: 4096
 
+--pkgmgr::
+   Package manager to use. Default: PackageManager from CCpp.conf
+   or "dnf"
+
 -e,--exact::
    Download only specified files
 
--R,--releasever::
```

Comment:
I can’t even tell why `--R` was here to begin with…

Reply:
Ah, `abrt-action-install-debuginfo-to-abrt-cache` accepts it.

Reply:
So I’ll chalk it up to being a genuine mistake, because `abrt-action-install-debuginfo-to-abrt-cache` actually does a conversion from `-R` to `--releasever`.